app/code/Magento/CatalogInventory/Api/StockQtyDecimalInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CatalogInventory\Api;
+
+/**
+ *  StockQtyDecimalInterface
+ */
+interface StockQtyDecimalInterface
```

Comment:
This interface looks more like SPI than API, it's better to place it in the Model namespace